src/api/routes/v2/schemas.ts

```diff
@@ -0,0 +1,70 @@
+import { Type, Static } from '@sinclair/typebox';
+import { TypeCompiler } from '@sinclair/typebox/compiler';
+import { ResourceType, pagingQueryLimits } from '../../../api/pagination';
+
+// ==========================
+// Parameters
+// ==========================
+
+const OffsetParam = Type.Integer({
+  minimum: 0,
+  title: 'Offset',
+  description: 'Result offset',
+});
+
+export const BlockLimitParam = Type.Integer({
+  minimum: 1,
+  maximum: pagingQueryLimits[ResourceType.Block].maxLimit,
+  default: pagingQueryLimits[ResourceType.Block].defaultLimit,
+  title: 'Block limit',
+  description: 'Blocks per page',
+});
+
+const BurnBlockHashParam = Type.RegExp(/^[0]{8}[a-fA-F0-9]{56}$/, {
```

Comment:
The `0` prefix check here should be removed. The number of leading zeros in a bitcoin block hash is dependent on network difficulty, which changes overtime on mainnet, and is completely different on testnet and regtest.

Reply:
Done